code-studio/src/js/levelbuilder_gamelab.js

```diff
@@ -0,0 +1,21 @@
+/** @file JavaScript run only on the gamelab level edit page. */
+import $ from 'jquery';
+import initializeCodeMirror from './initializeCodeMirror';
+import {throwIfSerializedAnimationListIsInvalid} from '@cdo/apps/gamelab/PropTypes';
+
+$(document).ready(function () {
+
+  // Live-validate animations JSON using same validation code we use in Gamelab
+  let levelStartAnimationsValidationDiv = $('#level-start-animations-validation');
+  initializeCodeMirror('level_start_animations', 'json', codeMirror => {
```

Comment:
There's an amazing `jsonEditor` table thing we could set up in place of codemirror, but this was the easiest to get up and running for now (and is a whole lot better than no immediate feedback).  Also, I'm not sure the `jsonEditor` integration is worth it if we're going to do a more robust animation-specific editor later.